roles/container_runtime/tasks/registry_auth.yml

```diff
@@ -18,3 +18,22 @@
   retries: 3
   delay: 5
   until: crt_oreg_auth_credentials_create is succeeded
+
+- name: Create for any additional registries
+  docker_creds:
+    path: "{{ docker_cli_auth_config_path }}"
+    registry: "{{ item.host }}"
+    username: "{{ item.user | default('openshift') }}"
+    password: "{{ item.password }}"
+    # Test that we can actually connect with provided info
+    test_login: "{{ oreg_test_login | default(True) }}"
+    proxy_vars: "{{ l_docker_creds_proxy_vars }}"
+    image_name: "{{ l_docker_creds_image_name }}"
+  when:
+  - openshift_additional_registry_credentials is defined
+  register: crt_oreg_auth_credentials_create
```

Comment:
Reusing registered variable names, while not problematic at this case (AFAICT), can create problems.  Don't do that.